aten/src/ATen/native/cuda/jit_utils.h

```diff
@@ -44,15 +46,23 @@ NvrtcFunction jit_pwise_function(
 
 void launch_jitted_pwise_function(
     NvrtcFunction function,
-    std::array<void*, 7>& args,
+    void* args[],
```

Comment:
Have updated the type of `args` from `std::array` to `void* []` (we don't have to allocate larger `std::array` than the actual number of args). 
This was also one of the reasons for limiting the numbers of additional arguments previously.